kong/llm/drivers/gemini.lua

```diff
@@ -643,6 +652,9 @@ function _M.configure_request(conf, identity_interface)
                                                              or "generateContent"
   local f_url = model.options and model.options.upstream_url
 
+  -- remove the `google/` prefix if it exists
+  local model_name = strip_provider_prefix(model.name)
```

Comment:
I don't have a context for this. How is the model name used in the plugin? Will it be used to construct requests? If so, will altering the name change the behavior?

Reply:
I have reverted my last commit. This PR only changes the `json` decode to `json.safe` to avoid returning 500 when the response body is empty.